cli/ballerina-tool/src/main/java/org/ballerinalang/tool/util/ToolUtil.java

```diff
@@ -118,7 +123,7 @@ private static void setCurrentBallerinaVersion(String version) throws IOExceptio
      */
     private static String getCurrentToolsVersion() {
         //TODO: Need to read folder
-        return "1.0.0-beta";
+        return "1.0.0-beta-SNAPSHOT";
```

Comment:
Can we get this from a system property or env ?

Reply:
We have to do something like that once we separate tool jar from ballerina lang, until that we might need at least read a file and get the tool version.